### PR TITLE
desktop/nwg-displays: Updated for version 0.3.13.

### PR DESCRIPTION
**Pull request by mac-a-r0ni**

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build desktop/nwg-displays

File: desktop/nwg-displays/nwg-displays.SlackBuild [hunk 1]
```diff
@@ -83,10 +83,8 @@ find -L . \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
 python3 setup.py install --optimize=1 --root=$PKG
-mkdir -p $PKG/usr/share/pixmaps $PKG/usr/share/applications
```
Comment (willysr) [inline desktop/nwg-displays/nwg-displays.SlackBuild:86]: you need to undelete this line as it creates the directory before you can copy something into it

Comment (mac-a-r0ni) [reply desktop/nwg-displays/nwg-displays.SlackBuild:86]: Fixed.

**Comment by willysr** [comment]
@sbo-bot: build desktop/nwg-displays

**Comment by willysr** [comment]
@mac-a-r0ni this still failed

**Comment by willysr** [comment]
@sbo-bot: build desktop/nwg-displays

**Comment by willysr** [comment]
merged

**Pull request closed without merging**